instance/instance.go

```diff
@@ -109,6 +109,66 @@ func (s *Store) Add(w http.ResponseWriter, r *http.Request) {
 	log.Debug("add instance", log.Data{"instance": instance})
 }
 
+// UpdateDimension updates label and/or description for a specific dimension within an instance
+func (s *Store) UpdateDimension(w http.ResponseWriter, r *http.Request) {
+
+	vars := mux.Vars(r)
+	ID := vars["id"]
+	dimension := vars["dimension"]
+
+	// Get instance
+	instance, err := s.GetInstance(ID)
+	if err != nil {
+		log.Error(err, nil)
+	}
+
+	// Early return if instance is already published
+	if instance.State == "published" {
+		w.WriteHeader(http.StatusForbidden)
+		return
+	}
+
+	// Read and unmarshal request body
+	bytes, err := ioutil.ReadAll(r.Body)
+	if err != nil {
+		log.Error(err, nil)
+	}
+
+	var dim *models.CodeList
+
+	err = json.Unmarshal(bytes, &dim)
+	if err != nil {
+		log.Error(err, nil)
+	}
+
+	// Update instance-dimension
+	for i := 0; i < len(instance.Dimensions); i++ {
```

Comment:
Could use the `range` operator here -https://github.com/golang/go/wiki/Range

e.g `for i := range instance.Dimensions`